.github/workflows/template-janitor.yml

```
@@ -41,17 +46,17 @@ jobs:
           sed -i "s/myproject/${{ github.event.repository.name }}/gi" CMakeLists.txt configured_files/config.hpp.in
 
           # Update URL placeholders for project
-          sed -i "s/%%myurl%%/${{ github.event.repositoryUrl }}/gi" CMakeLists.txt
+          sed -i "s|%%myurl%%|${{ env.NEW_URL }}|gi" CMakeLists.txt
```

```suggestion
          sed -i "s|%%myurl%%|${{ fromJson(steps.get_repo_meta.outputs.data).html_url }}|gi" CMakeLists.txt
```

Just noticed that the repositoryUrl from the env contains the git link (git://github.com/cpp-best-practices/cpp_boilerplate_project.git). See action runs.

maybe this one is what realy needs to go here? See https://docs.github.com/en/rest/reference/repos#get-a-repository

---

haha, thanks for the back and forth here.

There really are very many different moving pieces, and things that are difficult to test when it comes to CMake.

Example: the `option`s were recently broken and it went unnoticed for a time.